### PR TITLE
fix: Deletion of comments

### PR DESCRIPTION
**Pull request by orronai**
- Comments which deleted are not going to be appeared after showing the popover again and again

### CODE REVIEW

File: lms/static/grader.js [hunk 1]
```diff
@@ -52,12 +52,18 @@ function sendComment(kind, fileId, line, commentData) {
 function visuallyRemoveComment(commentId) {
```
Comment (yammesicka) [inline lms/static/grader.js:52]: Maybe we should also remove the trash icon from comments that aren't belong to the user?

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: lms/static/grader.js (hunk 1 above)
Comment (orronai) [reply lms/static/grader.js:52]: Only staff are going to see all the trashes

Comment (orronai) [reply lms/static/grader.js:52]: Users will see their trash only

Comment (orronai) [reply lms/static/grader.js:52]: Look at `isSolverComment` function in `comments.js`

**Review by yammesicka — APPROVED** [review]

**Comment by orronai** [comment]
Fixes #212

**Pull request merged**